.github/workflows/robustness-nightly.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       count: 80
       testTimeout: 200m
       artifactName: main-arm64
-      runs-on: "['self-hosted', 'Linux', 'ARM64']"
+      runs-on: actuated-arm64-8cpu-32gb
```

Comment:
Good catch from @serathius.  I believe for robustness this needs to be `"['actuated-arm64-8cpu-32gb']"`, refer line `18` above for example.

We can probably simplify the `fromJson` that the robustness nightly template does after this pr merges because the array will no longer be present.

Reply:
Don't think `"['actuated-arm64-8cpu-32gb']"` is the correct solution as Github Actions would treat `actuated-arm64-8cpu-32gb` as a label and not machine type. I think the proposed value makes sense assuming we will remove `fromJson`, but I'm not sure. It's a matter of trial and error.